### DISCUSSION TITLE
Is "override settings" no more supported?

### DISCUSSION THREAD

**Discussion by NottMee** [mifi/lossless-cut#2280] [General]
Firstly thanks for the great software.

In Readme it is said that settings can be overrided with this command for example:
**LosslessCut --settings-json '{captureFormat:"jpeg", "keyframeCut":true}'**

I tried that command, LosslessCut just runs in background without GUI.

Then I tried
**LosslessCut --settings-json '{captureFormat:"jpeg"}'**
The GUI comes but the capture format is not changed to jpeg

Is "override settings" no more supported?
If it is still supported, what did I do wrong?
How can I e.g. override the Output Dir? ( in config.json "customOutDir": "D:\\\\Test\\\\" )
Thanks for the answers in advance

Version: 3.64
OS: Win11

**Comment by mifi** [comment]
Hi. I think on windows you might have to quite it differently (i asked chatgpt):
```
LosslessCut.exe --settings-json "{\"captureFormat\":\"jpeg\", \"keyframeCut\":true}"
```

**Reply by NottMee** [reply]
Thanks for the tip. It works now perfectly. I also asked ChatGPT before, but somehow he didn't give me the right solution on Windows 😀
